File: Algorithm/Baekjoon/Python/조건문/주사위 세개.py
i = input().split()
j = []
for a in i:
    j.append(int(a))
s = set(j)

if len(s) == 1:
    print(10000 + j[0] * 1000)
elif len(s) == 2:
    if j[0] == j[1]:
        print(1000 + j[0] * 100)
    elif j[0] == j[2]:
        print(1000 + j[0] * 100)
    else:
        print(1000 + j[1] * 100)
else:
    print(100 * max(j))

File: Algorithm/Baekjoon/Java/조건문/주사위_세개.java
package Algorithm.Baekjoon.Java.조건문;

import java.util.*;

public class 주사위_세개 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        String[] numbers = sc.nextLine().split(" ");
        int a = Integer.parseInt(numbers[0]);
        int b = Integer.parseInt(numbers[1]);
        int c = Integer.parseInt(numbers[2]);

        int[] n = { a, b, c };
        Arrays.sort(n);

        if (a != b && b != c && c != a) {
            System.out.println(100 * n[2]);
        } else if (a == b && b == c && c == a) {
            System.out.println(10000 + 1000 * n[2]);
        } else {
            if (a == b) {
                System.out.println(1000 + 100 * a);
            } else if (b == c) {
                System.out.println(1000 + 100 * b);
            } else if (a == c) {
                System.out.println(1000 + 100 * a);
            }
        }
        sc.close();
    }
}
